Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -1210,77 +1257,31 @@ ModelExperimental.prototype.update = function (frameState) {
   }
 
   const context = frameState.context;
-  const referenceMatrix = defaultValue(this.referenceMatrix, this.modelMatrix);
-
-  // Update the image-based lighting for this model to detect any changes in parameters.
-  this._imageBasedLighting.update(frameState);
-
-  if (
-    this._imageBasedLighting.useSphericalHarmonicCoefficients ||
-    this._imageBasedLighting.useSpecularEnvironmentMaps
-  ) {
-    let iblReferenceFrameMatrix3 = scratchIBLReferenceFrameMatrix3;
-    let iblReferenceFrameMatrix4 = scratchIBLReferenceFrameMatrix4;
+  this._defaultTexture = context.defaultTexture;
```

Comment:
If we're reorganizing `update()`, why don't we split this into a bunch of small functions so you can tell at a glance the order of operations?

e.g. similar to how `PrimitiveStatisticsPipelineStage.process()` is just a list of function calls